### PR TITLE
add Founder rank and replaces freeze with editmute,

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java [hunk 1]
```diff
@@ -152,13 +158,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
...
             player.setGameMode(GameMode.SURVIVAL);
-            plugin.pl.getPlayer(player).getFreezeData().setFrozen(true);
+            plugin.pl.getPlayer(player).setEditBlocked(true);
             player.sendMessage(ChatColor.RED + "You are marked as an impostor, please verify yourself!");
             return;
         }
 
         // Set display
-        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()))
+        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()) || FUtil.FOUNDER.contains(player.getName()))
```
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java:167]: Why are you adding the founder setting here?

**Review by Wild1145 — CHANGES_REQUESTED** [review]
This again needs to be split into two separate PR's, the founder part merged with the rest of the founder changes, and the other changes as its own PR.

**Comment by Wild1145** [comment]
Thank you for contributing to the project, could you please amend your opening message to include the issue that this PR resolves. Thank You.

File: src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java (hunk 1 above)
Comment (ghost) [reply src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java:167]: To dont bug impostering system

**Pull request closed without merging**